code/game/objects/effects/decals/contraband.dm

```diff
@@ -362,6 +365,35 @@
 	desc = "The Griffin commands you to be the worst you can be. Will you?"
 	icon_state = "poster33"
 
+/obj/structure/sign/poster/contraband/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/toy/crayon/red/syndicate))
+		to_chat(usr, "<span class = 'notice'>There is no reason to change the message of this poster.</span>")
+		return
+	else
+		..()
+
+/obj/structure/sign/poster
+	var/corrupted_icon
+	var/corrupted_desc
+	var/corrupted_name
+
+/obj/structure/sign/poster/official/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/toy/crayon/red/syndicate))
+		var/obj/item/toy/crayon/red/syndicate/C = I
+		visible_message("<span class = 'notice'>You start drawing over the [src] with your crayon.</span>", "<span class = 'notice'>[user] starts drawing over the [src].</span>")
```

Comment:
This will make the poster do a visible message since `src` is implied.
The params are also as follows `visible_message(message, self_message, blind_message)`
You switched the two around here.